### DISCUSSION TITLE
How can I add Event Annotations to the roasting curve through button during coffee roasting?

### DISCUSSION THREAD

**Discussion by ghoslin01** [artisan-roaster-scope/artisan#1684] [Q&A]
I need to record the ROR, temperature, and DTR at key points during the roasting process. Could you please let me know if it's possible to record this information with a single button press? thanks

**Comment by poundy** [comment]
Do you have Artisan recording your roasting data already? What do you need more than that? Can you not look at the recorded profile and see what you need? I suspect you'll get more meaningful answers if you fully explain your scenario and what you're hoping to achieve
